src/common/res/features/pacing/main.js

```diff
@@ -4,113 +4,150 @@
-		  var budgeted = available+activity;
-		  var burned = activity/budgeted;
-		  var pace = burned/timeSpent();
-		
-		  var masterName = $.trim($(this).prevAll('.is-master-category').first().find('.budget-table-cell-name').text());
-		  var subcatName = $.trim($(this).find('.budget-table-cell-name').text());
-		
-		  var transactionCount = allTransactions.filter((el) => el.transferAccountId == null 
-		    && el.outflow > 0 && el.subCategoryNameWrapped == (masterName+": "+subcatName)).length;
-		
-		  var displayType = 'dollars';
-		  if(displayType == 'percentage') {
-		    if(pace > 1) {
-		      var temperature = 'warm cautious';
-		    } else {
-		      var temperature = 'cool positive';
-		    }
-		    if(!isFinite(pace)) {
-		      var display = 999;
-		    } else {
-		      var display = Math.max(0,Math.round(pace*100));
-		    }
-		    if(pace > 0) {
-		      $(this).append('<li class="budget-table-cell-available budget-table-cell-pacing"><span class="budget-table-cell-pacing-display '+temperature+'">'+display+'%</span></li>');
-		    }
-		  } else if (displayType == 'dollars') {
-		    display = Math.round((budgeted*timeSpent()-activity)*1000);
-		    if(available != 0 && activity != 0 && masterName != 'Credit Card Payments') {
-		      if(pace > 1) {
-		        var temperature = 'warm cautious';
-		      } else if(activity != 0) {
-		        var temperature = 'cool positive';
-		      } else {
-		        var temperature = 'neutral';
-		      }
-		    } else {
-		      var temperature = 'neutral'; 
-		    }
-		
-		    if(display >= 0) {
-		      var tooltip = 'In '+transactionCount+' transaction'+(transactionCount != 1 ? 's' : '')+' you have spent '+ formatCurrency(display, false)+
-							' less than your available budget for this category '+Math.round(timeSpent()*100)+'% of the way through the month.';
-		    } else if(display < 0) {
-		      var tooltip = 'In '+transactionCount+' transaction'+(transactionCount != 1 ? 's' : '')+' you have spent '+ formatCurrency(-display, false)+
-							' more than your available budget for this category '+Math.round(timeSpent()*100)+'% of the way through the month.';
-		    }
-		    $(this).append('<li class="budget-table-cell-available budget-table-cell-pacing"><span title="'+tooltip+'" class="budget-table-cell-pacing-display '+temperature+'">'+ formatCurrency(display, true)+'</span></li>');
-		  }
-		});
-		
-
-	};
-	ynabToolKit.insertPacingColumns();
-
-
-    
+      function parseSelectedMonth() {
+        // TODO: There's probably a better way to reference this view, but this works better than DOM scraping which seems to fail in Firefox
+        if($('.ember-view .budget-header').length) {
+          var headerView = Ember.View.views[$('.ember-view .budget-header').attr("id")];
+          var endOfLastMonth = headerView.get("currentMonth").toNativeDate();
+          return new Date(endOfLastMonth.getFullYear(), endOfLastMonth.getMonth()+1, 1);
```

Comment:
Are you shure about +1? It shows me Feb when I'm in Jan. I removed it in ynabToolKit object.